### DISCUSSION TITLE
How to run multiple magento instances using the same docker

### DISCUSSION THREAD

**Discussion by abdulbasitnawab** [markshust/docker-magento#870] [General]
Hi
If i have multiple magento projects is it possible to run them via same docker container or each app will have a different one, with a different url.

**Comment by markshust** [comment]
Yes, this repo works just fine with that. Just bin/stop, bin/start on the respective site you'd like to boot down and start up, and you can run as many development environment as you wish. I typically run at least 10 sites at different times with different versions of Magento and PHP using this setup.
